easybuild/easyconfigs/c/CUDA/CUDA-7.0.28.eb

```diff
@@ -0,0 +1,16 @@
+name = 'CUDA'
+version = '7.0.28'
+
+homepage = 'https://developer.nvidia.com/cuda-toolkit'
+description = """CUDA (formerly Compute Unified Device Architecture) is a parallel
+ computing platform and programming model created by NVIDIA and implemented by the
+ graphics processing units (GPUs) that they produce. CUDA gives developers access
+ to the virtual instruction set and memory of the parallel computational elements in CUDA GPUs."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+source_urls = ['http://developer.download.nvidia.com/compute/cuda/%(version_major_minor)s/Prod/local_installers/']
```

Comment:
@thiell for 7.0.x, they use `7_0` rather than `7.0` in the download URL...

so, replace replace with:

```python
source_urls = ['http://developer.download.nvidia.com/compute/cuda/%(version_major)s_%(version_minor)s/Prod/local_installers/']
```

or (since the above is too long, strictly speaking):

```python
source_urls = [
    'http://developer.download.nvidia.com/compute/cuda/%(version_major)s_%(version_minor)s/Prod/local_installers/',
]
```